infra/infra-integrations/infra-integrations-email/src/main/java/com/sflpro/notifier/externalclients/email/mandrill/MandrillConfiguration.java

```diff
@@ -0,0 +1,45 @@
+package com.sflpro.notifier.externalclients.email.mandrill;
+
+import com.microtripit.mandrillapp.lutung.controller.MandrillMessagesApi;
+import com.sflpro.notifier.email.TemplatedEmailSenderRegistry;
+import com.sflpro.notifier.externalclients.email.mandrill.communicator.MandrillApiCommunicator;
+import com.sflpro.notifier.externalclients.email.mandrill.communicator.MandrillApiCommunicatorImpl;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.Assert;
+
+
+/**
+ * Created by Hayk Mkrtchyan.
+ * Date: 6/19/19
+ * Time: 11:39 AM
+ */
+@Configuration
+@ConditionalOnProperty(name = MandrillConfiguration.TOKEN_PLACE_HOLDER)
+class MandrillConfiguration {
```

Comment:
Ideally, from the perspective of a person who implements a `NotificationServiceProvider`, I would prefer to know as little about the notifier platform as possible. Ideally, I will have to implement just one single Interface. No other code at all that has to be aware of the other components of the notifier.

Reply:
Done!